apps/crn-server/src/entities/research-output.ts

```diff
@@ -115,6 +115,7 @@ export const parseGraphQLResearchOutput = (
         id: group.id,
         title: group.flatData.title || '',
       })) || [],
+    published: true,
```

Comment:
This is the part that is hard coded for now that I've mentioned in the description of the PR. This is part of another ticket [Shared Output: Display Draft](https://asaphub.atlassian.net/browse/CRN-1304)